src/components/ContainerDetailsSubheader.react.js

```diff
@@ -99,7 +99,7 @@ var ContainerDetailsSubheader = React.createClass({
   },
   handleTerminal: function () {
     if (!this.disableTerminal()) {
-      metrics.track('Terminaled Into Container');
+      metrics.track('Terminated Into Container');
```

Comment:
Although it looks like a typo, this is actually correct, as we opened a 'terminal into the container', hence the use of 'Terminaled' as a verb